source/kubernetes/appendix.rst

```diff
@@ -105,6 +105,29 @@ Available cluster labels
      - Maximum number of worker nodes to scale out to, if auto scaling is enabled.
 
        This value is required if ``auto_scaling_enabled`` is ``true``.
+   * - ``auto_healing_enabled``
+     - Boolean
+     - ``true``
+     - ``true`` or ``false``
+     - Enable auto-healing on control plane and worker nodes.
+
+       With auto-healing enabled, if nodes become NotReady for an extended duration they will be
```

Comment:
Don't feel strongly about this one, I personally think this looks nicer though.

```suggestion
       With auto-healing enabled, if nodes become ``NotReady`` for an extended duration they will be
```

Reply:
Done